<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(194, 194, 194);
</string>
  </property>
  <widget class="QPushButton" name="spinbutton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>780</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
font-size: 15pt;</string>
   </property>
   <property name="text">
    <string>↻</string>
   </property>
  </widget>
  <widget class="QLabel" name="imagelabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>1121</width>
     <height>611</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backbutton">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>930</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
font-size: 20pt;</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="selabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>920</y>
     <width>761</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15pt;
color: red;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="rlabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>852</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>R:</string>
   </property>
  </widget>
  <widget class="QLabel" name="glabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>852</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>G:</string>
   </property>
  </widget>
  <widget class="QLabel" name="blabel">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>852</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>B:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>885</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15pt;</string>
   </property>
   <property name="text">
    <string>의심 확장자 목록</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rlineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>852</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="glineEdit">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>852</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="blineEdit">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>852</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>852</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
font-size: 15pt;</string>
   </property>
   <property name="text">
    <string>CHANGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>780</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
font-size: 15pt;</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>780</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
font-size: 15pt;</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>30</y>
     <width>751</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20pt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
